src/rsz/src/RepairDesign.hh

```diff
@@ -250,6 +251,12 @@ protected:
   static constexpr float elmore_skew_factor_ = 1.39;
   static constexpr int min_print_interval_ = 10;
   static constexpr int max_print_interval_ = 100;
+
+  // best slew numbers to ensure the max_slew in SDC is reasonable
+  float best_case_slew_;
+  float best_case_slew_load_;
+  bool best_case_slew_computed_;
+
```

Comment:
Harkening back to my comment above I think it would be good to encapsulate these into their own class.

Reply:
I will create a new file. As I mentioned earlier, since these checks are dependent on loads, doing them too early might not be the best idea. On the other hand, the infinite loop of inserting a buffer at the same spot  had two root causes. One of them is the slew limit and the other one had to do with the bug in logic where we are splitting a net of length L into a net of length L and a subnet of length 0.